app_dart/test/request_handlers/refresh_chromebot_status_test.dart

```diff
@@ -7,22 +7,47 @@ import 'package:cocoon_service/src/model/appengine/task.dart';
 import 'package:cocoon_service/src/request_handlers/refresh_chromebot_status.dart';
 import 'package:cocoon_service/src/service/datastore.dart';
 import 'package:cocoon_service/src/service/luci.dart';
+import 'package:github/server.dart';
 import 'package:mockito/mockito.dart';
 import 'package:test/test.dart';
 
 import '../src/datastore/fake_cocoon_config.dart';
 import '../src/request_handling/api_request_handler_tester.dart';
 import '../src/request_handling/fake_authentication.dart';
+import '../src/request_handling/fake_http.dart';
+import '../src/service/fake_github_service.dart';
+
+const String branchRegExp = '''
+      master
+      ^flutter-[0-9]+\.[0-9]+-candidate\.[0-9]+
+      ''';
 
 void main() {
   group('RefreshChromebotStatus', () {
     FakeConfig config;
     ApiRequestHandlerTester tester;
     MockLuciService mockLuciService;
     RefreshChromebotStatus handler;
+    List<String> githubBranches;
+    FakeHttpClient branchHttpClient;
+
+    Stream<Branch> branchStream() async* {
```

Comment:
Branch stream seems to be used through the tests. We should move it to a test utils file to remove the duplication through the codebase.

Reply:
This is a good point. At least five APIs are using this code. Created a TODO to simply test codes, and prefer to work on it in a separate PR.

Reply:
Tracking issue: https://github.com/flutter/flutter/issues/53280